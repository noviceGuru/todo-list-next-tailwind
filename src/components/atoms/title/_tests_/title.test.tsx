import { render, screen } from '@testing-library/react'

import Title from '../title'

it('renderes title correctly with the correct looks', () => {

	const tableRow = document.createElement('tr')
	render(
		<Title
			title='dummy text title to test'
		/>
		, { container: document.body.appendChild(tableRow) })
	
	const title = screen.getByRole('row')
	expect(title.textContent).toBe('dummy text title to test')
	expect(title).toMatchSnapshot()
})