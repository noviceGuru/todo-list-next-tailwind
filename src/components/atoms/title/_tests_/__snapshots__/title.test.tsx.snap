// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderes title correctly with the correct looks 1`] = `
<tr>
  <span
    class=" flex pt-2 pl-3"
  >
    dummy text title to test
  </span>
</tr>
`;
